src/sas/sascalc/fit/pagestate.py

```diff
@@ -490,7 +490,12 @@ def _repr_helper(self, list, rep):
         Helper method to print a state
         """
         for item in list:
-            rep += "parameter name: %s \n" % str(item[1])
+            if str(item[1][-5:]) == 'width':
+                par = str(item[1][:-6])
```

Comment:
Please check that it works with a squarewell model, that has the parameter `wellwidth`.  You probably want to match against `'.width'` rather than `'width'` to protect against this.  Couldn't test since I don't know how to trigger the code.

Reply:
Thanks Paul. You were right, there was an error with the squarewell model. Changed the comparison to '.width'. To test, make a fit with at least 1 PD parameter and call Report results. Previous output was e.g.: radius.width = 0.1 +- 0.03. Now it will be radius.width (gaussian) = 0.1 +- 0.03.
Question for Richard: Do you think it is worthy to add also the number of points and sigmas? E.g.:
radius.width (gaussian, points=35, sigmas=3) = 0.1 +- 0.03.